cellprofiler/modules/maskobjects.py

```diff
@@ -41,22 +57,9 @@
 -  *Parent:* The label number of the parent object.
 -  *Location_X, Location_Y:* The pixel (X,Y) coordinates of the center
    of mass of the masked objects.
-"""
-
-import cellprofiler.measurement
-import numpy as np
-import scipy.ndimage as scind
-from centrosome.cpmorphology import fixup_scipy_ndimage_result as fix
-from centrosome.outline import outline
-
-import cellprofiler.image as cpi
-import cellprofiler.module as cpm
-import cellprofiler.measurement as cpmeas
-import cellprofiler.object as cpo
-import cellprofiler.preferences as cpprefs
-import cellprofiler.setting as cps
-import identify as I
-from cellprofiler.setting import YES, NO
+""".format(**{
+    "HELP_ON_SAVING_OBJECTS": _help.HELP_ON_SAVING_OBJECTS
```

Comment:
There should be a "See also" section header with the `HELP_ON_SAVING_OUTLINES` text before the section "Measurements made by this module".

Reply:
Did you mean help_on_saving_objects?

Reply:
Ah, yes!

Reply:
Sounds good!